DataManagementSystem/Client/DataManager.py

```diff
@@ -582,6 +582,37 @@ def replicate( self, lfn, destSE, sourceSE = '', destPath = '', localCache = ''
       return res
     return S_OK( lfn )
 
+  def __getSERealName( self, storageName ):
```

Comment:
Well, this is questionable: if a method is only called at one place, why create an artificial dependency? I agree it can be a function and not a method... Will do the change...